### ISSUE TITLE
Aliased imports don't show the true imported component

### ISSUE THREAD

**Issue by JDansercoer** [moroshko/react-scanner#10]
When a component is imported like

```
import { Icon as DesignSystemIcon } from '@my/design-system'
```

the generated output is

```
{
  "DesignSystemIcon": {
    "instances": 1,
    "props": {
      "color": 1,
      "flex": 1
    }
  }
}
```

Is there a possibility to get the actual name of the imported component, and not the Alias it was imported as?

**Comment by moroshko** [comment]
Yeah, makes sense.

If one file has:

```
import { Button } from "design-system";
```

and in another file you decided to alias the `Button` for some reason:

```
import { Button as MyButton } from "design-system";
```

you'd want them to go to the same bucket, right?

```
{
  Button: {
    instances: [
      { ... },
      { ... }
    ]
  }
}
```

Would knowing the alias `MyButton` still be valuable?

```
{
  Button: {
    instances: [
      { ... },
      {
        importAlias: "MyButton",
        ...
      }
    ]
  }
}
```

**Comment by JDansercoer** [comment]
Well sure, the more info the better I guess!

**Comment by sapegin** [comment]
Yup, knowing the alias is useful: if some component is often aliased, it might suggest that the name is conflicting with other dependencies on the project.

**Comment by moroshko** [comment]
This is fixed in 0.5.0. 
Also, added `importInfo` to component instances. You can use it to find aliases.
